.github/workflows/main.yml

```diff
@@ -28,33 +27,37 @@ jobs:
           - 5432:5432
     steps:
     - uses: actions/checkout@v2
-    - run: |
-        apt-get update
-        apt-get install --yes --no-install-recommends postgresql-client
     - name: Connect to PostgreSQL
       run: |
-        PGPASSWORD=postgres psql -h postgres -c 'CREATE DATABASE bit_schema_test;' -U postgres
-        PGPASSWORD=postgres psql -h postgres -c 'CREATE SCHEMA bitschema;' -U postgres -d bit_schema_test
-        PGPASSWORD=postgres psql -h postgres -c 'CREATE SCHEMA test_schema;' -U postgres -d bit_schema_test
-        PGPASSWORD=postgres psql -h postgres -c 'create SCHEMA test_schema_2;' -U postgres -d bit_schema_test
-        PGPASSWORD=postgres psql -h postgres -c '\dn;' -U postgres -d bit_schema_test
-        PGPASSWORD=postgres psql -h postgres -c 'show search_path;' -U postgres -d bit_schema_test
-        PGPASSWORD=postgres psql -h postgres -c "ALTER DATABASE bit_schema_test SET search_path TO bitschema,public;" -U postgres -d bit_schema_test
-        PGPASSWORD=postgres psql -h postgres -c 'show search_path;' -U postgres -d bit_schema_test
+        PGPASSWORD=postgres psql -h localhost -c 'CREATE DATABASE bit_schema_test;' -U postgres
+        PGPASSWORD=postgres psql -h localhost -c 'CREATE SCHEMA bitschema;' -U postgres -d bit_schema_test
+        PGPASSWORD=postgres psql -h localhost -c 'CREATE SCHEMA test_schema;' -U postgres -d bit_schema_test
+        PGPASSWORD=postgres psql -h localhost -c 'create SCHEMA test_schema_2;' -U postgres -d bit_schema_test
+        PGPASSWORD=postgres psql -h localhost -c '\dn;' -U postgres -d bit_schema_test
+        PGPASSWORD=postgres psql -h localhost -c 'show search_path;' -U postgres -d bit_schema_test
+        PGPASSWORD=postgres psql -h localhost -c "ALTER DATABASE bit_schema_test SET search_path TO bitschema,public;" -U postgres -d bit_schema_test
+        PGPASSWORD=postgres psql -h localhost -c 'show search_path;' -U postgres -d bit_schema_test
       env:
-        POSTGRES_HOST: postgres
+        POSTGRES_HOST: localhost
         POSTGRES_PORT: 5432
     - name: Set up Python 3.6
       uses: actions/setup-python@v2
       with:
         python-version: 3.6
     - name: Install dependencies
       run: |
-        python -m pip install --upgrade pip
+        sudo -H python -m pip install --upgrade pip
+        pip --version
         if [ -f requirements.txt ]; then pip install -r requirements.txt; fi
     - name: Run tests
       run: |
         python -m unittest discover tests -v
+      env:
+        DB_PASSWORD: postgres 
+        DB_TYPE: postgres 
```

Comment:
@epicadk , I think this is the culprit why the tests failed as it's missing `psycopg2` as pointed out by the error message.
<img width="727" alt="Screen Shot 2021-03-05 at 4 04 37 pm" src="https://user-images.githubusercontent.com/29667122/110070486-36240f80-7dce-11eb-8383-262a2f157d5a.png">

Can you see if adding the `+psycopg2` trail (so DB_TYPE: postgresql+psycopg2) fix this? That's the DB_TYPE I used on the local env 
<img width="539" alt="Screen Shot 2021-03-05 at 4 06 48 pm" src="https://user-images.githubusercontent.com/29667122/110070562-5f44a000-7dce-11eb-8e51-07d8f9858b1f.png">


and running the tests locally pass.
<img width="746" alt="Screen Shot 2021-03-05 at 3 59 28 pm" src="https://user-images.githubusercontent.com/29667122/110070536-50f68400-7dce-11eb-9c08-fb232b9983ae.png">

Reply:
@mtreacy002 Oops I totally missed this. Would you like to to try using psycopg2? The tests are passing now

Reply:
I'll do that before pushing to upstream 😉